ducktape/command_line/main.py

```diff
@@ -184,5 +186,6 @@ def main():
     reporter = HTMLSummaryReporter(test_results)
     reporter.report()
 
+    update_latest_symlink(args.results_root, results_dir)
```

Comment:
My thinking here was that any error that propagates to `main` (probably causing reports not to be generated) or which takes place in `main`  is catastrophic enough that it doesn't make sense to ensure the symlink points to the new report.